jnigen/test/kotlin_test/lib/kotlin.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
```

Comment:
Should have a more specific name than kotlin, maybe `kotlin_suspend_fun.dart`?

(also 2023)

Reply:
It's gonna have more Kotlin related things in the future (if we decide to keep it in single_file mode).